Sources/Core/Sources/Render/Shader/ChunkShaders.metal

```diff
@@ -89,3 +89,38 @@ fragment float4 chunkFragmentShader(RasterizerData in [[stage_in]],
 
   return color;
 }
+
+
+constant float2 quadVertices[] = {
+     float2(-1.0,  1.0),
+     float2(-1.0, -1.0),
+     float2( 1.0, -1.0),
+     float2(-1.0,  1.0),
+     float2( 1.0,  1.0),
+     float2( 1.0, -1.0)
+ };
+
+ struct QuadVertex {
+     float4 position [[position]];
+     float2 uv;
+ };
+
+
+ fragment float4 screenFragmentFunction(QuadVertex v [[stage_in]],
+                                  texture2d <float> outputImage [[ texture(0) ]]) {
+
+     constexpr sampler smplr(coord::normalized);
+     float4 cellColour = outputImage.sample(smplr, v.uv);
+     return cellColour;
+ };
+
+
+ vertex QuadVertex screenVertexFunction(uint id [[vertex_id]]) {
+
+     auto quadVertex = quadVertices[id];
```

Comment:
You're a pro, I never knew you could just use `auto`, I only learnt that you could do that in cpp pretty recently lol

Reply:
MSL is ❤️